### DISCUSSION TITLE
Enterprise Roundup December 2024

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#146925] [Enterprise]
![Header of Enterprise category features the text Enterprise Roundup on the upper lefthand corner on a black background with an abstract peach and teal abstract line design](https://github.com/user-attachments/assets/7f286656-ba58-49ef-857a-4bd9ea7e4c4c)

As we wind down 2024 and enter such a festive period, we want to check in one last time to say thank you for being a part of the Enterprise category and community. The Enterprise category was a new addition to the community this year and thanks to you, we’ve been able to share knowledge, uplevel our GitHub skills, and improve our features. 

### ⭐ Delightful Discussions: These were some of the top posts of 2024 
* **GHAS Certification Exam Prep - Part One**: This was the first week of our guided study course for the GitHub Advanced Security Preparation series. Good news! This content is always available so you can start studying at any time beginning with [this discussion](https://github.com/orgs/community/discussions/137197).  
* **Git Environments - Ability to Save As and create a Duplicate Environment**: [The feedback discussion](https://github.com/orgs/community/discussions/120102) to add a "Save As" feature to duplicate Git Environments welcomed a ton of different perspectives. Check them out and feel free to add your own or upvote. 
* **GitHub Enterprise Server - VMware ESXi v8**: Our very own @djdefi answered this question in [the discussion](https://github.com/orgs/community/discussions/123616). 

### 🗳️ Recent Releases: All updates, info, releases, etc that are Enterprise Server and Cloud -related!
* **Enterprise repository properties, policies and rulesets – Public Preview**: This preview includes enterprise custom repository properties, enterprise repository policies and enterprise rulesets to help enterprise administrators manage more at greater scale. Share questions, feedback, and comments in [our discussion](https://github.com/orgs/community/discussions/145599). 
* **GitHub Enterprise Server 3.15 is now generally available**: This newest version gives customers enhanced deployment requirements and security controls. See what’s new and share your thoughts in [this discussion](https://github.com/orgs/community/discussions/144554). 
* **Migrate repositories with GitHub Enterprise Importer using GitHub owned blob storage – Private Preview**: You can now enroll in a private preview to use GitHub-owned storage when migrating repositories to GitHub Enterprise Cloud using GitHub Enterprise Importer (GEI). Find all the details in [the changelog](https://github.blog/changelog/2024-11-21-migrate-repositories-with-github-enterprise-importer-using-github-owned-blob-storage-private-preview/). 

### 📣 Announcements: The latest updates around the Community and GitHub 🆕
* ❗**GitHub Incident Updates**: As of December 5, incidents will now be posted in the community. [Read our announcement](https://github.com/orgs/community/discussions/146469). 
* **Git your learning on with GitHub Copilot Certification**: We’re in our last week of the four-week study guide to help you prepare. [Join us in this Discussion](https://github.com/orgs/community/discussions/144443) and the course to explore essential AI topics, engage in discussions, and try hands-on exercises that’ll make you exam-ready. 💚

We hope your end of year is whatever you need it to be: restful, merry, or bright. We look forward to seeing you back here in 2025 ✨.

**Comment by Salgado2004** [comment]
What a ride! 2024 was a great year, full of innovation and discovery! It was a great pleasure to get to know more about GitHub and study for certifications 😁🚀 The exam prep series for GitHub Advanced Security and GitHub Copilot were amazing! I'm really glad to be part of this 💞 
Waiting 2025...

**Reply by queenofcorgis** [reply]
It was so great to see all that you accomplished in the community this year, we've got even more planned for 2025 😄

**Comment by queenofcorgis** [comment]
Our first check-in of 2025 has been [posted in a new discussion](https://github.com/orgs/community/discussions/149742), get caught up and inspired!
